CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## v1.1dev - [date]
 
+* remove spaces in genome headers and replace special nt by N in hairpin file for mirdeep2 to work. Follow up from [[#69]] (https://github.com/nf-core/smrnaseq/pull/79)
```

Comment:
```suggestion
* remove spaces in genome headers and replace special nt by N in hairpin file for mirdeep2 to work. [[#69]] (https://github.com/nf-core/smrnaseq/pull/79)
```

Reply:
I am sorry I am realizing I have not removed spaces from the genome file headers
it was the following command you suggested
`perl -plane 's/\s+.+$//' < $refgenome > $refgenome`
if you want I can do it now